code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -62,6 +62,7 @@
 	new /obj/item/storage/box/flashbangs(src)
 	new /obj/item/shield/riot/tele(src)
 	new /obj/item/storage/belt/security/full(src)
+	new /obj/item/flashlight/seclite(src)
```

Comment:
This is a balance change, not a fix, because now it'll apply to all five maps instead of just Tram. Either revert this change or remark your chanelog.

Reply:
done